src/api/app/controllers/webui/request_controller.rb

```diff
@@ -149,6 +149,12 @@ def beta_show
     @diff_limit = params[:full_diff] ? 0 : nil
     @diff_to_superseded_id = params[:diff_to_superseded]
     @actions = @bs_request.webui_actions(filelimit: @diff_limit, tarlimit: @diff_limit, diff_to_superseded: @diff_to_superseded, diffs: false)
+    reviews = @bs_request.reviews.where(state: 'new')
+    @my_open_reviews = reviews.select { |review| review.matches_user?(User.session) }
```

Comment:
I think his can be moved to `BsRequest` as a parameterized scope

Reply:
I'll do it outside of this PR to keep the scope of this PR limited